index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css"/>
+    </head>
```

Comment:
Kindly add some of the meta elements missing in the head section. For example, you can add a 'charset' attribute.